docs/contents/components/forms/select/index.mdx

```diff
@@ -73,6 +73,32 @@ To set a default value, set a string to `defaultValue`.
 </Select>
 ```
 
+### Header
+
+```tsx
+<Select
+  placeholder="キャラクターを選択"
+  header={<header>Header added here</header>}
```

Comment:
Change

```jsx
header={<header>Header added here</header>}
```

to

```jsx
header={<Center pt="2" px="3">Header added here</Center>}
```

Reply:
[f411e3d](https://github.com/yamada-ui/yamada-ui/pull/2634/commits/f411e3d2a53f9a4c4c189a520e4215a58511cb6e)